Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -99,6 +99,10 @@ CClientGame::CClientGame(bool bLocalPlay)
     m_lastWeaponSlot = WEAPONSLOT_MAX;            // last stored weapon slot, for weapon slot syncing to server (sets to invalid value)
     ResetAmmoInClip();
 
+    m_pSurfaceInfo = reinterpret_cast<CSurfaceType*>(0xB79538);
+    // Store default surface properties
+    memcpy(&m_pOriginalSurfaceInfo, (void *)0xB79538, sizeof(CSurfaceType));
```

Comment:
Instead of working with raw memory in the deathmatch module you could move this to game_sa or "port" the surface properties from the data file into C++.